README.md

```diff
@@ -0,0 +1,26 @@
+# E-mail_Parser
+Tool to calculate overtime worked through sent e-mails timestamp. 
+
+## Installing
+Install the required packages:
+
+```
+pip3 install -r requirements.txt
+```
+#### Outlook
+You need to disable cached exchange mode in order to get all e-mais from server. You can see how in https://support.microsoft.com/
```

Comment:
Este link da Microsoft e' muito vago. Future-you will thank you if you're more specific.

Reply:
Done, fui preguiçoso. A pagina dava-me em português e como meti o repository public, não fazia sentido meter coisas em português. Já consegui encontrar a versão em inglês e atualizei.